NEWS.md

```diff
@@ -121,6 +118,12 @@ This release incorporates the following documentation enhancements:
 
     See [OpenSSL Guide].
 
+This release incorporates the following bug fixes and mitigations:
+
+  * Fixed excessive time spent in DH check / generation with large Q parameter
+    value
+    [(CVE-2023-5678)]
```

Comment:
Other CVEs seem to be of the following form - what am I missing?

```suggestion
    ([CVE-2023-5678])
```

Reply:
Heh, yeah, this one is wrong. @mattcaswell please correct it.